housingbayarea/bloom
# feat: creating new SMC ami charts

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/3588)


## Description
This creates 2 new ami charts for smc outlined in attached issue

I also created a new script (heavily based on the existing one) that is closer.

I think once we switch to prisma adding a new migration to do this each time will be difficult so the new script still allows for migrations, but is closer to what the openapi docs page would also want to accept

Also we now have some examples of what the input/output of the script is 

## How Can This Be Tested/Reviewed?
Run the migration locally and you should see the 2 new ami charts get created.
these new charts should be selectable when adding units to a listing

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
